doc/Makefile.in

```diff
@@ -122,19 +122,19 @@ XML_SOURCES = \
 	performance_tips.xml \
 	postgis.xml \
 	reference_accessor.xml \
+	reference_bbox.xml \
 	reference_constructor.xml \
 	reference_editor.xml \
 	reference_guc.xml \
 	reference_lrs.xml \
 	reference_management.xml \
 	reference_measure.xml \
 	reference_sfcgal.xml \
-	reference_misc.xml \
 	reference_operator.xml \
 	reference_output.xml \
 	reference_processing.xml \
 	reference_raster.xml \
-	reference_temporal.xml \
```

Comment:
This is a historic thing. ST_ initially is SpatioTemporal, but by time to design the spec for temporal writers got tired and just said "ok let it be SpaTial". @estebanzimanyi you know the terminology about time, can you advise here?

Reply:
The only functions under Temporal were ones dealing with Trajectories, so it seems better to give them a section by that name.  That highlights the ability to work with trajectories.

Reply:
As for the ST_ naming, maybe that reflected grand ambitions at one time.  But AFAIK the specs haven't really fulfilled these dreams, so now it is just a (nice?) way to namespace spatial functions.